crux-metrics/src/crux/metrics/dropwizard.clj

```diff
@@ -45,3 +46,18 @@
 
 (defn meter-count [^Metered t]
   (.getCount t))
+
+;; Reporters
+
+(defn stop-reporter
```

Comment:
seems unused, given you return a Closeable from start-reporter? (this is the pattern elsewhere - good to see it being used here too)